ImportBlocks/ImportBlocks.cs

```diff
@@ -17,23 +17,14 @@ public override void Configure()
             Settings.Load(GetConfiguration());
         }
 
-        private bool OnExport(string[] args)
+        private void WriteBlocks(uint start, uint count, string path, bool writeStart)
         {
-            if (args.Length < 2) return false;
-            if (!string.Equals(args[1], "block", StringComparison.OrdinalIgnoreCase)
-                && !string.Equals(args[1], "blocks", StringComparison.OrdinalIgnoreCase))
-                return false;
-            if (args.Length >= 3 && uint.TryParse(args[2], out uint start))
+            uint end = start + count - 1;
+            using (FileStream fs = new FileStream(path, FileMode.OpenOrCreate, FileAccess.ReadWrite, FileShare.None, 4096, FileOptions.WriteThrough))
```

Comment:
Can you add a comment on these options?

Reply:
* `FileOption.WriteThrough` indicates that the system should write through any intermediate cache and go directly to disk.
* `4096` is the default size of `buffer`.